__tests__/intergration.test.js

```diff
@@ -141,4 +141,51 @@ describe("/api/articles/:article_id",()=>{
             expect(body.msg).toBe("Error 400 - Bad Request Given")
         })
     })
+})
+
+//GET ALL COMMENTS BY ARTICLE ID======================
+
+describe("/api/articles/:article_id/comments",()=>{
+    test("GET: 200 - Received all columns from comments depending on article ID",()=>{
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body:{comments}})=>{
```

Comment:
Here we can auto-pass if the array is empty. Within the then block and before the forEach we must write an expect statement asserting that the array's length is greater than zero